### PR TITLE
Add new openapi gen

### PR DESCRIPTION
**Pull request by obsd**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,12 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "update-openapi.json": "curl https://api.permit.io/v2/openapi.json -o openapi.json",
```
Comment (RazcoDev) [inline package.json:9]: Will it work with the `.json` in the `update-openapi.json` ?

File: package.json [hunk 2]
```diff
@@ -5,7 +5,12 @@
...
+    "update-api": "nswag run nswag.json",
+    "update-status-check": "pwd; sed -i -f update_status_check.sed PermitPDPOpenAPI.cs",
```
Comment (RazcoDev) [inline package.json:11]: 😎

File: src/permit/Enforcement.cs [hunk 3]
```diff
@@ -1,27 +1,30 @@
...
+using PermitSDK.Models;
+using PermitSDK.PDP.OpenAPI;
 
 public class PermissionCheckException : Exception
 {
     public PermissionCheckException() { }
 
-    public PermissionCheckException(string message) : base(message) { }
+    public PermissionCheckException(string message)
+        : base(message) { }
```
Comment (RazcoDev) [inline src/permit/Enforcement.cs:19]: Just making sure, this new line here is fine ?

**Review by RazcoDev — APPROVED** [review]
Great PR, waiting for the other APIs

File: package.json (hunk 1 above)
Comment (obsd) [reply package.json:9]: not sure I understand

File: src/permit/Enforcement.cs (hunk 3 above)
Comment (obsd) [reply src/permit/Enforcement.cs:19]: it complied and I have a test that uses this files (the .Net SDK tests are good, saved my **s few times :)

**Pull request merged**